t/support/files/inc_dir_1-1/dummydata1-1

```diff
@@ -0,0 +1 @@
+dummydata1-1
```

Comment:
Is this just to test a backup? Can you reuse the dummy files in t/support/ already?